### PR TITLE
Fixing warn log message

### PR DESCRIPTION
**Pull request by pgbhagat**
Fixing warn log message

### CODE REVIEW

**Comment by pgbhagat** [comment]
I signed it!

File: google-cloud-translate/src/main/java/com/google/cloud/translate/TranslateOptions.java [hunk 1]
```diff
@@ -169,7 +169,7 @@ private TranslateOptions(Builder builder) {
       logger.log(
           Level.WARNING,
           String.format(
-              "Ignoring Application Default Credentials: using explicit setting for API key instead.",
+              "Ignoring Application Default Credentials %s: using explicit setting for API key instead.",
```
Comment (pongad) [inline google-cloud-translate/src/main/java/com/google/cloud/translate/TranslateOptions.java]: Hmm. Can we instead do
```java
logger.log(Level.WARNING, "Ignoring Application Default Credentials {}:...", ServiceOptions.CREDENTIAL_ENV_NAME);
```
? That way we don't need to format the string if the log is turned off.

Comment (pgbhagat) [reply google-cloud-translate/src/main/java/com/google/cloud/translate/TranslateOptions.java]: Done.

File: google-cloud-translate/src/main/java/com/google/cloud/translate/TranslateOptions.java [hunk 2]
```diff
@@ -155,30 +155,28 @@ private TranslateOptions(Builder builder) {
...
+                "Ignoring API key set in environment variable {}: using explicit setting for credentials instead.",
+                API_KEY_ENV_NAME);
       }
     } else if (builder.apiKey != null) {
       credentials = null;
       apiKey = builder.apiKey;
       logger.log(
           Level.WARNING,
-          String.format(
-              "Ignoring Application Default Credentials: using explicit setting for API key instead.",
```
Comment (pgbhagat) [inline google-cloud-translate/src/main/java/com/google/cloud/translate/TranslateOptions.java]: String.format was missing %s earlier for arg  ServiceOptions.CREDENTIAL_ENV_NAME.

**Comment by pongad** [comment]
@pgbhagat Gah, I'm sorry. The format string should have been `{0}`, not `{}`. I was thinking in Python. LGTM otherwise. I apologize for the churn.

**Comment by pgbhagat** [comment]
No problem, i checked other files and it is used like {0}. Corrected in the next commit.

**Comment by pgbhagat** [comment]
Is the AppVeyor build is failing because of
'Build execution time has reached the maximum allowed time for your plan (60 minutes).' ?

**Pull request merged**